package.json

```diff
@@ -3,11 +3,16 @@
   "version": "1.0.1",
   "description": "Timers with near-infinite duration support",
   "main": "index.js",
+  "files": [
+    "index.js",
+    "component.json"
```

Comment:
component.json is for GitHub installs only (it's a dead build system anyway, so perhaps it's time to remove the file altogether... but that's out of scope for this PR), so it doesn't have to be included in the NPM package.